classes/controller/AdminController.php

```diff
@@ -638,6 +643,12 @@ public function processFilter()
             }
 
             foreach ($_GET as $key => $value) {
+
+                // Handle forced filtering parameter by url
+                if (stripos($key, 'list_idFilter_') === 0) {
+                    $key = str_replace('list_id', $this->list_id, $key);
```

Comment:
We should ensure that only the first occurrence of `list_id` is replaced. Just in case somebody called
```php
$link->getAdminLink($c, true, [], [ 'my_list_id' => 100 ]);
```

https://www.designcise.com/web/tutorial/how-to-replace-only-the-first-occurrence-of-a-string-in-php